README.rst

```diff
@@ -90,14 +94,15 @@ and overrides the ``configure()`` method to define all of the application compon
 
   public class WebAnalyticsApp extends AbstractApplication {
     static final String APP_NAME = "WebAnalyticsApp";
+    static final String APP_DESCRIPTION = "Data Analysis with an OLAP Cube";
```

Comment:
I don't like description in guide. It only makes code looks more verbose, pollutted without adding value..